src/views/AmalgamationRegular/ReviewConfirm.vue

```diff
@@ -101,6 +101,7 @@
         class="mt-6"
```

Comment:
I don't like that this component's name is `IncorporationDateTime` as it's being used in amalgamations now too. I was thinking of renaming it. Thoughts?

Reply:
Agreed.

Not only that, but this component is in the incorporation file folder (not common).

Do you have an idea for the new name? "filingDateTime" perhaps?

Reply:
Yes, that could work. I will be doing that then!

Reply:
Or "effectiveDateTime.vue" ?

Reply:
Went with `EffectiveDateTime` 🙌

Reply:
LGTM